### PR TITLE
SJE-02 about-us

### PR DESCRIPTION
**Pull request by Bubblegumi**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,17 @@
 <!-- main screen finish -->
 
 <!-- about us start -->
+    <section class="about-us" id="about-us">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12 text-center">
+                    <div class="header-label "><h2>Про нас</h2></div>
```
Comment (mosiichuk) [inline src/index.html:18]: Remove extra space `header-label`

File: src/styles/style.sass [hunk 2]
```diff
@@ -62,6 +62,28 @@ a
...
+    
+    
+.header-label 
+    background-color: $accent-orange
+    width:  346px
+    height: 100%
+    margin: 0 auto
+    height: 10%
+    
+
```
Comment (mosiichuk) [inline src/styles/style.sass:79]: Remove extra new line. Should be 1 instead of 2.

File: src/styles/style.sass [hunk 3]
```diff
@@ -62,6 +62,28 @@ a
...
+.about-desc
+    max-width: 620px
+    text-align: justify
+    margin: 0 auto
+    padding-bottom: 16px
+
+
```
Comment (mosiichuk) [inline src/styles/style.sass:86]: Remove extra new line. Should be 1 instead of 2.

File: src/styles/style.sass [hunk 4]
```diff
@@ -62,6 +62,28 @@ a
 // <!-- main screen finish -->
 
 // <!-- about us start -->
+.about-us 
+    padding-top: 35px
+    background-color: $text-white
+    background: url('../img/about-bg.png') center top / cover
+    width: 100%
+    
+
```
Comment (mosiichuk) [inline src/styles/style.sass:71]: Remove extra new line. Should be 1 instead of 2.

File: src/styles/style.sass [hunk 5]
```diff
@@ -62,6 +62,28 @@ a
 // <!-- main screen finish -->
 
 // <!-- about us start -->
+.about-us 
+    padding-top: 35px
+    background-color: $text-white
+    background: url('../img/about-bg.png') center top / cover
+    width: 100%
```
Comment (mosiichuk) [inline src/styles/style.sass:69]: Please remove `width: 100%` section and div by default have 100% width.

File: src/styles/style.sass [hunk 6]
```diff
@@ -62,6 +62,28 @@ a
...
+    
+    
+.header-label 
+    background-color: $accent-orange
+    width:  346px
+    height: 100%
+    margin: 0 auto
+    height: 10%
+
```
Comment (mosiichuk) [inline src/styles/style.sass:78]: Remove `height: 100%` and `height: 10%`, instead add padding-bottom and padding-top

File: src/styles/style.sass [hunk 7]
```diff
@@ -62,6 +62,28 @@ a
...
+    height: 100%
+    margin: 0 auto
+    height: 10%
+    
+
+.about-desc
+    max-width: 620px
+    text-align: justify
+    margin: 0 auto
+    padding-bottom: 16px
```
Comment (mosiichuk) [inline src/styles/style.sass:84]: Move `padding-bottom` to `about-us` class instead.

**Comment by Bubblegumi** [comment]
image

**Pull request merged**